### DISCUSSION TITLE
User Roles and PhotoSync (Photo viewing apps)

### DISCUSSION THREAD

**Discussion by HowdyMcGee** [photoprism/photoprism#4377] [Q&A] [answered]
At the end of the day, my goal is to have 1 user managing photos and other users able to view those photos via mobile apps (such as PhotoSync or Live Photos). My understanding is that I would need to purchase a Plus license to get access to user roles.

What's not clear is how the apps work with User roles if I don't want those users to upload.

Does PhotoPrism user roles work well with the suggested apps like PhotoSync and Live Photos? What are my options?

**Comment by graciousgrey** [comment, marked as answer]
Users with the roles Guest or Viewer cannot upload files, neither within PhotoPrism nor with a sync app like PhotoSync. 

https://docs.photoprism.app/user-guide/users/roles/
